.db.env

```diff
@@ -1,28 +1,25 @@
 # PostgreSQL
-TEST_POSTGRES_BASE_URI="postgres://prisma:prisma@localhost:5432"
 TEST_POSTGRES_URI="postgres://prisma:prisma@localhost:5432/tests"
-# Note: the isolated instance is only needed for one test (client/src/__tests__/integration/errors/connection-limit-postgres/test.ts)
-TEST_POSTGRES_ISOLATED_URI="postgres://prisma:prisma@localhost:5435/tests"
 TEST_POSTGRES_URI_MIGRATE="postgres://prisma:prisma@localhost:5432/tests-migrate"
 TEST_POSTGRES_SHADOWDB_URI_MIGRATE="postgres://prisma:prisma@localhost:5432/tests-migrate-shadowdb"
+# Note: the isolated instance is only needed for one test (client/src/__tests__/integration/errors/connection-limit-postgres/test.ts)
+TEST_POSTGRES_ISOLATED_URI="postgres://prisma:prisma@localhost:5435/tests"
 
 # MySQL
-TEST_MYSQL_BASE_URI="mysql://root:root@localhost:3306"
 TEST_MYSQL_URI="mysql://root:root@localhost:3306/tests"
-# Note: the isolated instance is only needed for one test (client/src/__tests__/integration/errors/connection-limit-mysql/test.ts)
-TEST_MYSQL_ISOLATED_URI="mysql://root:root@localhost:3307/tests"
 TEST_MYSQL_URI_MIGRATE="mysql://root:root@localhost:3306/tests-migrate"
 TEST_MYSQL_SHADOWDB_URI_MIGRATE="mysql://root:root@localhost:3306/tests-migrate-shadowdb"
+# Note: the isolated instance is only needed for one test (client/src/__tests__/integration/errors/connection-limit-mysql/test.ts)
+TEST_MYSQL_ISOLATED_URI="mysql://root:root@localhost:3307/tests"
 
 # MariaDB
-TEST_MARIADB_BASE_URI="mysql://root:root@localhost:4306"
-TEST_MARIADB_URI="mysql://prisma:prisma@localhost:4306/tests"
+TEST_MARIADB_URI="mysql://root:root@localhost:4306/tests"
```

Comment:
```suggestion
# Note that the database name must be `tests` (in our test setup we search and replace that value later)
TEST_MARIADB_URI="mysql://root:root@localhost:4306/tests"
```

Reply:
Decided to add a note at top of file instead as otherwise we need to repeat this one too many times all over the files